UPGRADE-1.10.md

```diff
@@ -1,3 +1,24 @@
+# UPGRADE FROM `v1.10.x` TO `v1.10.12`
+1. Order Processors' priorities have changed and `sylius.order_processing.order_prices_recalculator` has now a higher priority than `sylius.order_processing.order_shipment_processor`.
+
+Previous priorities:
+```shell
+sylius.order_processing.order_adjustments_clearer          60         Sylius\Component\Core\OrderProcessing\OrderAdjustmentsClearer   
+sylius.order_processing.order_shipment_processor           50         Sylius\Component\Core\OrderProcessing\OrderShipmentProcessor    
+sylius.order_processing.order_prices_recalculator          40         Sylius\Component\Core\OrderProcessing\OrderPricesRecalculator   
+...     
+```
+
+Current priorities:
+```shell
+sylius.order_processing.order_adjustments_clearer          60         Sylius\Component\Core\OrderProcessing\OrderAdjustmentsClearer   
+sylius.order_processing.order_prices_recalculator          50         Sylius\Component\Core\OrderProcessing\OrderPricesRecalculator   
+sylius.order_processing.order_shipment_processor           40         Sylius\Component\Core\OrderProcessing\OrderShipmentProcessor    
+...     
+```
+
+If you rely on previous priorities, you can bring them back by setting flag ``processing_shipments_before_prices`` to ``true`` in ``config/packages/_sylius.yaml``. However, it is not recommended because new priorities fix [invalid estimated shipping costs](https://github.com/Sylius/Sylius/pull/13769).
```

Comment:
Can we add sample parameters config here? I would solve issues with the proper key before this one (`sylius_core`)

Reply:
Done, thanks :slightly_smiling_face: